concepts.tex

```diff
@@ -127,19 +127,23 @@
 
 {\color{newclr}
 \enterexample
-The following type \tcode{T} satisfies the explicitly stated syntactic requirements
-of concept \tcode{C} above but does not satisfy the additional implicit requirements:
+The following type \tcode{T} \oldtxt{satisfies}\newtxt{meets} the explicitly stated syntactic requirements
+of concept \tcode{C} above but does not \oldtxt{satisfy}\newtxt{meet} the additional implicit requirements:
 
 \begin{codeblock}
 struct T {
-  bool operator==(T const &) const { return true; }
-  bool operator==(T &) = delete;
+  bool operator==(@\oldtxt{T const \&}\newtxt{const T\&}@) const { return true; }
+  bool operator==(@\oldtxt{T \&}\newtxt{T\&}@) = delete;
 };
 \end{codeblock}
 
-Since \tcode{T} fails to satisfy the implicit requirements of \tcode{C}, \tcode{C<T>()} is not
-satisfied in principle. Whether it is satisfied in practice is undefined.
-\exitexample
+\oldtxt{Since }\tcode{T} fails to \oldtxt{satisfy}\newtxt{meet} the implicit
+requirements of \tcode{C}, so \tcode{C<T>()} is not satisfied\oldtxt{ in principle}.
+\oldtxt{Whether it is satisfied in practice is undefined.} \newtxt{Since
+implementations are not required to validate the syntax of implicit requirements, it
+is unspecified whether or not an implementation diagnoses a program which requires
+\tcode{C<T>()} as ill-formed.}
+ \exitexample
```

Comment:
> "... a program which requires `C<T>()` as ill-formed"

That's a strange and awkward construction. "... a program that requires that `C<T>()` to be ill-formed"? And what does it mean that a concept check is ill-formed? That it might cause a hard error at compile time? I don't think that's what we want to say.